sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -172,8 +172,6 @@ export interface SearchRequest {
   captions?: QueryCaptionType;
   /** The comma-separated list of field names used for semantic search. */
   semanticFields?: string;
-  /** The query parameters for vector and hybrid search queries. */
-  vector?: Vector;
```

Comment:
No updates to the API report?

Reply:
This client isn't exposed to users. We just coerce the public type to this one for the REST API requests.